src/js/rx/components/ConfirmRefillModal.jsx

```diff
@@ -25,13 +24,10 @@ class ConfirmRefillModal extends React.Component {
     // Initialize to prevent console errors
     let innerElement = '';
 
-    if (this.props.isLoading) {
-      innerElement = (
-        <LoadingIndicator
-          message="Submitting your refill request..."/>
-      );
-    } else if (this.props.prescription) {
+    if (this.props.prescription) {
```

Comment:
There's not a situation where isLoading is true and there's no prescription prop?

Reply:
Good question, this was just what I determined from looking at the logic around it. The previous loader's message was "submitting your refill request" so I would think not but I'll see if @U-DON can confirm.

Reply:
I don't think it should occur, since `isLoading` becomes true only upon making a refill request. It does make me a little uneasy about what edge cases we might be missing. Like perhaps we shouldn't be showing a modal at all if there's no prescription currently being viewed.